core/tests/webdriverio_utils/forms.js

```diff
@@ -323,24 +323,32 @@ var UnicodeEditor = function(elem) {
 };
 
 var AutocompleteDropdownEditor = function(elem) {
-  var containerLocator = '.select2-container';
-  var searchInputLocator = '.select2-search input';
-  var dropdownElement = '.select2-dropdown';
+  var containerLocator = '.e2e-test-exploration-category-dropdown';
+  var searchInputLocator = '.mat-select-search-input.mat-input-element';
+  var categorySelectorChoice = '.e2e-test-exploration-category-selector-choice';
+
   return {
     setValue: async function(text) {
       await action.click('Container Element', elem.$(containerLocator));
       await action.waitForAutosave();
       // NOTE: the input field is top-level in the DOM, and is outside the
       // context of 'elem'. The 'select2-dropdown' id is assigned to the input
       // field when it is 'activated', i.e. when the dropdown is clicked.
+      var searchInputLocatorText = function(text) {
+        return $(`.e2e-test-exploration-category-selector-choice=${text}`);
+      };
+
       await action.setValue(
-        'Dropdown Element',
-        $(dropdownElement).$(searchInputLocator),
-        text + '\n');
+        'Dropdown Element Search', $(searchInputLocator), text);
+      await action.click(
```

Comment:
Why we need to click on the option to select it if this can be done using `selectByVisibleText` which we were using earlier in webdriverio?

Reply:
removed